sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1881,8 +1919,10 @@ if used in application code.
 
 . Using reductions in a graph node.
 . Using sycl streams in a graph node.
-. Profiling an event returned from graph submission with
-  `event::get_profiling_info()`.
+. Profiling information is not available for graphs that contain host-task nodes.
```

Comment:
This restriction seems like another good reason not to expose per-node profiling information.  I don't understand why the presence of a host task would impact the runtime's ability to reason about a different device kernel, and I think many users would struggle to understand this too.

It's also unclear to me whether the presence of a host task disables _all_ profiling.  What is the intent here?  Is it still possible to use the event returned by the graph submission to reason about how long the entire graph took to execute?